installing_gitops/installing-openshift-gitops.adoc

```diff
@@ -14,7 +14,7 @@ toc::[]
 * You have access to the {OCP} web console.
 * You are logged in as a user with the `cluster-admin` role.
 * You are logged in to the {OCP} cluster as an administrator.
-* Your cluster has the link:https://docs.openshift.com/container-platform/latest/installing/cluster-capabilities.html#marketplace-operator_cluster-capabilities[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
+* Your cluster has the link:https://docs.openshift.com/container-platform/latest/installing/overview/cluster-capabilities.html#marketplace-operator_cluster-capabilities[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
```

Comment:
```suggestion
*You have either enabled the `marketplace` capability or manually configured the Red Hat Operator catalog source. For more information, see "link:https://docs.openshift.com/container-platform/latest/installing/overview/cluster-capabilities.html#marketplace-operator_cluster-capabilities[Marketplace capability]"
```

- Using `marketplace` capability to keep it consistent with the wording in the "Marketplace capability" section.
- Using "You have" to keep it consistent with the other items in the list.

Reply:
@Dhruv-Soni11 since the suggested changes by @sr1kar99 above are not in scope of this PR, feel free to take your call regarding implementing the suggestion. This is not a mandatory change.

Reply:
I would not like to fix this comment.